version.go

```diff
@@ -1,3 +1,3 @@
 package main
 
-var version = "0.3.0"
+var version = "0.3.1"
```

Comment:
Oh, now I'm wondering, isn't it "CSI Spec Compatibility" version instead of driver's itself version?

Reply:
It is the version of the driver itself.
You add a new functionality not performing a fix, so 0.4.0 will be better.

Reply:
Thanks. Changed version to "0.4.0".